NSS-Day-Cohort-53/tabloid-full-stack-always-picturesque-icebergs
# Delete Category

#### Changes Made
1. Added file `DeleteCategory.js` to `client` directory.
1. Modified file `CategoryRepository.cs` to include `getCategoryById` and DeleteCategory.
2. Modified file `ICategoryRepository.cs` to include `getCategoryById` and DeleteCategory signatures.
3. Modified file `CategoryController.cs` to include `Delete` and functionality.
4. Modified file `CategoryManager.js` to include `getCategoryById` and DeleteCategory.
5. Modified file `ApplicationView.js` to include route for delete category component.
6. Modified file `Category.cs` to include `a delete button for the category.

#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout iw-deleteCategory
    ```
7. Open a new Terminal tab (⌘T) and navigate to the server directory.
8. Test app functionality.
    > click on categories in nav bar
    > see list of categories, select the delete button next to one of the categories
    > be redirected to deletion confirmation page
    > Select delete button, be redirected back to previous page observe deleted category
        - select cancel and will return you to previous page without deleting category
9. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
